chain/challenges-10.test.js

```diff
@@ -0,0 +1,198 @@
+
+'use strict';
```

Comment:
'use strict' should be the first line. (Not a big deal, but wanted to point it out!)